tests/test_basic_read.py

```diff
@@ -0,0 +1,54 @@
+"""Tests for the basic_read function in the csvy library."""
+
+import unittest
+from pathlib import Path
+
+from csvy.readers import basic_read
+
+
+class TestBasicRead(unittest.TestCase):
```

Comment:
`PyCSVY` uses `pytest` for writing the tests, not `unittest`. Please, adapt this to use the right test suite. See the [test_readers.py](https://github.com/ImperialCollegeLondon/pycsvy/blob/develop/tests/test_read.py) file to get an idea of how this is done.